api/src/main/kotlin/com/ford/internalprojects/peoplemover/space/Space.kt

```diff
@@ -39,12 +36,6 @@ data class Space (
     @Column(unique = true, nullable = false)
     var name: String,
 
-    @OneToMany(mappedBy = "spaceId", orphanRemoval = true, cascade = [CascadeType.REMOVE, CascadeType.REFRESH], fetch = FetchType.EAGER)
-    val roles: Set<SpaceRole> = HashSet(),
-
-    @OneToMany(mappedBy = "spaceId", orphanRemoval = true, cascade = [CascadeType.REMOVE, CascadeType.REFRESH], fetch = FetchType.EAGER)
-    val locations: List<SpaceLocation> = ArrayList(),
-
```

Comment:
Did we ever replace these with one-to-many mappings for spaceUuid? I see in the repository layer we're now exposing a "findAllBySpaceUuid" function - is that a comprehensive replacement for getting rid of these annotations?

Reply:
We did not replace these and yes, we use findAllBySpaceUuid instead toe get the locations and roles associated with a space